Scripts/create_pods_and_versions_index.rb

```diff
@@ -19,9 +22,27 @@
     end
   end
 end
+STDERR.puts "Generated #{shards.count} shards"
```

Comment:
why `STDERR`? was/is this used for debugging?

Reply:
This is output that's displayed in the Netlify build logs. I think it's nice to keep it there.